test/probe/http-client.test.js

```diff
@@ -819,4 +820,44 @@ describe("probe http-client", function() {
             assert.strictEqual(this.metrics, undefined);
         });
     });
+
+    describe("chunked encoding", function() {
+        let realServer;
+        const port = 8000; //server.getHttpPort() + 1;
```

Comment:
This is a problem as it would fail tests if something is running on this port.

In #61 I found a way to have `nock` get streamed responses.